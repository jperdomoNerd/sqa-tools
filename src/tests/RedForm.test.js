import renderer from 'react-test-renderer';
import RedForm from '../components/RedForm';

describe('RedForm Component', () => {
  
  it('<RedForm /> renders correctly', () => {
    const tree = renderer.create(<RedForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
