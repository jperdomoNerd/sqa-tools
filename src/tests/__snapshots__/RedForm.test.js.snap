// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedForm Component <RedForm /> renders correctly 1`] = `
<form
  className="form form--red"
>
  <h3
    className="form__title"
  >
    Parameters available
  </h3>
  <div
    className="form__input-container"
  >
    <label
      className="form__label"
    >
      abanumber:
    </label>
    <input
      className="form__input"
      type="text"
    />
  </div>
  <div
    className="form__input-container"
  >
    <label
      className="form__label"
    >
      accountnumber:
    </label>
    <input
      className="form__input"
      type="text"
    />
  </div>
  <div
    className="form__input-container"
  >
    <label
      className="form__label"
    >
      address:
    </label>
    <input
      className="form__input"
      type="text"
    />
  </div>
</form>
`;
